support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -316,6 +346,8 @@ export function Checkout() {
 	const [billingPostcodeStateLoading, setBillingPostcodeStateLoading] =
 		useState(false);
 
+	const formRef = useRef<HTMLFormElement>(null);
+	const [, setIsFormValid] = useState(false);
```

Comment:
What's going on here? I've not seen a use case for `useState` like this before 🤔 ...where we don't define a variable, such as `isFormValid` as the first item in destructured array.

Reply:
Good shout, we don't use this. I was trying to find a way to get the `disable`/`enable` methods from PayPal - [but found a better way](https://github.com/guardian/support-frontend/pull/5875/files#diff-dbc5d74a5f52e0cc93a8185ea2825653897711f50a6af3029dd91a90bcdd145fR795-R806).